GitVersionCore/Configuration/Wizard/PickBranchingStrategy2Step.cs

```diff
@@ -0,0 +1,39 @@
+namespace GitVersion
+{
+    using System;
+    using System.Collections.Generic;
+
+    public class PickBranchingStrategy2Step : ConfigInitWizardStep
```

Comment:
The class names I have picked are terrible.. I agree. Will have another go at that tonight